### PR TITLE
fix: bug trex delete doesn't delete some packages

### PR DESCRIPTION
**Pull request by Atticus64**
Code to solution the bug #137

### CODE REVIEW

File: handlers/delete_package.ts [hunk 1]
```diff
@@ -26,13 +26,24 @@ export async function deletepackage(toDelete: string) {
...
       if (Packages.imports) {
-        const toDelete = STD.includes(haveVersion(pkg))
-          ? `${haveVersion(pkg)}/`
-          : haveVersion(pkg);
 
-        delete Packages.imports[toDelete];
+        if (haveSlash) {
+          toDelete = STD.includes(haveVersion(pkg))
+            ? `${haveVersion(pkg)}/`
+            : haveVersion(pkg);
```
Comment (ghaerdi) [inline handlers/delete_package.ts:38]: This lines of code are useless in that scope condition but we need it to parse `example@version(/)` to `example(/)`

**Review by ghaerdi — CHANGES_REQUESTED** [review]

File: handlers/delete_package.ts (hunk 1 above)
Comment (Atticus64) [reply handlers/delete_package.ts:38]: Okey in one moment I fix that

**Comment by Atticus64** [comment]
@ghaerdi this is a better form to handle verification? 

```ts
const includesVersion = STD.includes(haveVersion(pkg))

      if (Packages.imports) {
        if (includesVersion) toDelete = `${haveVersion(pkg)}/`
```

**Comment by ghaerdi** [comment]
@Atticus64 I see you removed the `haveVersion(pkg)` variant, without the `/`. Add it again for cases like `non_std@version` to `non_std`

**Comment by Atticus64** [comment]
> toDelete = STD.includes(haveVersion(pkg))
>             ? `${haveVersion(pkg)}/`
>             : haveVersion(pkg);

Okey, now i back to original version and add if statatements for the key

File: handlers/delete_package.ts [hunk 2]
```diff
@@ -26,13 +26,30 @@ export async function deletepackage(toDelete: string) {
     try {
       const pkg: string = toDelete?.trim();
       const Packages = (await getImportMap<importMap>())!;
+      const packageNames = Object.keys(Packages.imports)
+      const pack = packageNames.find(p => p.includes(toDelete))
+      const haveSlash = pack?.includes("/")
+      const includesVersion = STD.includes(haveVersion(pkg))
```
Comment (ghaerdi) [inline handlers/delete_package.ts:32]: Remove this unused const

**Review by ghaerdi — CHANGES_REQUESTED** [review]
It's looks almost good to me, just remove that line and I will approve it.

File: handlers/delete_package.ts (hunk 2 above)
Comment (Atticus64) [reply handlers/delete_package.ts:32]: Okey in one moment I remover it

**Comment by Atticus64** [comment]
@ghaerdi ready, I removed the constant

**Review by ghaerdi — APPROVED** [review]
**✅ LGTM**

**Pull request merged**